go-controller/extensions/pkg/node/node_windows.go

```diff
@@ -24,14 +57,76 @@ func NewNodeHandler(clientset kubernetes.Interface, nodeName string) (types.Node
 // Add function learns about a new node being added to the cluster
 // For a windows node, this means watching for all nodes and programming the routing
 func (n *nodeController) Add(node *kapi.Node) {
+	network, err := hcn.GetNetworkByID(n.networkID)
+	if err != nil {
+		log.Error(err)
+		return
+	}
+
+	nodeSubnet, ok := node.Annotations[extensionHostSubnet]
```

Comment:
@JocelynBerrendonner I believe this function will get called for the local node too, so check that `node`'s UUID doesn't match this node.

Reply:
Fixed in the latest iteration! The tunnel functionality needs to be between Windows nodes too. From the VXLAN configuration perspective, Windows and Linux nodes should behave pretty much the same way.